spec/system/organization_system_spec.rb

```diff
@@ -19,6 +19,7 @@
   context "while signed in as an organization admin" do
     let!(:store) { create(:storage_location) }
     before do
+      create(:ndbn_member)
```

Comment:
I usually prefer being specific about the data we're going to be looking for (in this case, full_name) in the page. Otherwise you risk getting into situations where the text you're looking for just happens to match with something else (or it's blank by accident, and *that* matches, or or or).